src/getVehicleType.js

```diff
@@ -9,9 +9,19 @@ export default function getVehicleType({ licensePlate, licenseState }) {
   return axios
     .get(url, {
       headers: {
-        'accept-language': 'en-US,en;q=0.9',
-        'user-agent':
-          'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.114 Safari/537.36',
+        Accept:
+          'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,*/*;q=0.8',
+        'Accept-Encoding': 'gzip, deflate, br',
+        'Accept-Language': 'en-US,en;q=0.5',
+        Connection: 'keep-alive',
+        Host: 'www.carfax.com',
+        'Sec-Fetch-Dest': 'document',
+        'Sec-Fetch-Mode': 'navigate',
+        'Sec-Fetch-Site': 'none',
+        'Sec-Fetch-User': '?1',
+        'Upgrade-Insecure-Requests': 1,
+        'User-Agent':
+          'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:108.0) Gecko/20100101 Firefox/108.0',
       },
     })
```

Comment:
This change was required for me to get this test passing locally, but CI still doesn't like it: https://github.com/josephfrazier/reported-web/actions/runs/3825532430/jobs/6508591443

I suspect carfax may be blocking github IPs...